src/library/Uno.Themes.WinUI.Markup/Theme/TextBox.cs

```diff
@@ -0,0 +1,207 @@
+						public static ResourceValue<Thickness> Focused => new("FilledTextBoxBorderThicknessFocused", true);
+					}
+				}
+
+				public static class Outlined
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxForeground")]
+						public static ResourceValue<Brush> Default => new("OutlinedTextBoxForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("OutlinedTextBoxForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxForegroundFocused")]
+						public static ResourceValue<Brush> Focused => new("OutlinedTextBoxForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("OutlinedTextBoxForegroundDisabled", true);
+					}
+
+					public static class PlaceholderForeground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxPlaceholderForeground")]
+						public static ResourceValue<Brush> Default => new("OutlinedTextBoxPlaceholderForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxPlaceholderForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("OutlinedTextBoxPlaceholderForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxPlaceholderForegroundFocused")]
+						public static ResourceValue<Brush> Focused => new("OutlinedTextBoxPlaceholderForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxPlaceholderForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("OutlinedTextBoxPlaceholderForegroundDisabled", true);
+					}
+
+					public static class BorderBrush
+					{
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxBorderBrush")]
+						public static ResourceValue<Brush> Default => new("OutlinedTextBoxBorderBrush", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxBorderBrushPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("OutlinedTextBoxBorderBrushPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxBorderBrushFocused")]
+						public static ResourceValue<Brush> Focused => new("OutlinedTextBoxBorderBrushFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "OutlinedTextBoxBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("OutlinedTextBoxBorderBrushDisabled", true);
+					}
+
+					public static class Typo
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "OutlinedTextBoxFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("OutlinedTextBoxFontFamily", true);
+
+						[ResourceKeyDefinition(typeof(FontWeight), "OutlinedTextBoxFontWeight")]
```

Comment:
Same here for FontWeight

Reply:
fixed

Reply:
same (https://github.com/unoplatform/Uno.Themes/pull/1049#discussion_r1271013703)

Reply:
done